src/core/mac/mac_frame.cpp

```diff
@@ -66,6 +66,17 @@ const char *Address::ToString(char *aBuf, uint16_t aSize) const
     return aBuf;
 }
 
+uint16_t Frame::ReadUint16(const uint8_t *aBuffer)
```

Comment:
It seems this function doesn't rely on `Frame` at all. Reading and writing integers in little endian byte order might also be useful in other part. Is it better to share these functions within the namespace `ot::Utils` or `ot::Encoding`.